<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <widget class="QRadioButton" name="click">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Click</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="chose">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Chose</string>
    </property>
   </widget>
   <widget class="QPushButton" name="open_img">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>open image</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="img_area">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>600</width>
      <height>500</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>598</width>
       <height>498</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="bottom_info">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>560</y>
      <width>600</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="des_img_area">
    <property name="geometry">
     <rect>
      <x>629</x>
      <y>59</y>
      <width>600</width>
      <height>500</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>598</width>
       <height>498</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="des_bottom_info">
    <property name="geometry">
     <rect>
      <x>629</x>
      <y>560</y>
      <width>600</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="des_open_img">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>open des img</string>
    </property>
   </widget>
   <widget class="QPushButton" name="select_ok">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>select</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionFile">
   <property name="text">
    <string>File</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
